proxy/src/bin/proxy.rs

```diff
@@ -96,7 +97,8 @@ async fn main() -> anyhow::Result<()> {
     let _sentry_guard = init_sentry(Some(GIT_VERSION.into()), &[]);
 
     info!("Version: {GIT_VERSION}");
-    ::metrics::set_build_info_metric(GIT_VERSION);
+    info!("Build_tag: {BUILD_TAG}");
+    ::metrics::set_build_info_metric(GIT_VERSION, BUILD_TAG);
```

Comment:
Same question about the `::` prefix applies than here: https://github.com/neondatabase/neon/pull/5576/files#r1368368223

Reply:
Um, it is not possible to remove `::` prefix here.
The reason is that `::` prefix is used to indicate that `metrics` is the external crate, not internal module `metrics` of `proxy` crate.
The `proxy` crate has its own internal module `metrics` that the name is the same as external crate `metrics`.